### DISCUSSION TITLE
How to use AnimationGroup to show images gradually?

### DISCUSSION THREAD

**Discussion by manimnewbee** [3b1b/manim#1450] [Q&A] [answered]
Hi, can anyone please advise how to make images show one after another when inside an AnimationGroup?
For instance the following code:

AnimationGroup(
                    *[ApplyMethod(large_image.become, small_image) for large_image, small_image in zip(large_images, small_images)],
                    lag_ratio=0.6
                )
)

Shows at first all the large images, then gradually transforms them to the small images. I would like it to show images only after the previous images started the process of transforming, but it doesn't work for me.

Even if I select lag_ratio=1.0, I see all the images in the beginning.

**Comment by TonyCrane** [comment, marked as answer]
I don't know how you used that. But I used like this and it works:
```python
class Code1405(Scene):
    def construct(self):
        mob = VGroup(*[
            Square(2) for i in range(3)
        ]).arrange(RIGHT)
        mob_ = VGroup(*[
            a.copy().scale(0.5) for a in mob
        ])
        self.add(mob)
        self.wait()
        self.play(
            AnimationGroup(*[
                Transform(a, b) 
                for a, b in zip(mob, mob_)
            ], lag_ratio=0.6)
        )
```

https://user-images.githubusercontent.com/44120331/121772207-96cda000-cba6-11eb-86e6-579239700f6b.mp4
